src/main/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/command/ToolsApiScannerInstaller.java

```diff
@@ -169,7 +169,8 @@ protected HttpUrl getDownloadUrl() throws BlackDuckIntegrationException {
     protected String downloadSignatureScanner(File scannerExpansionDirectory, HttpUrl downloadUrl, String localScannerVersion) throws IOException, IntegrationException, ArchiveException {
         BlackDuckRequestBuilder requestBuilder = new BlackDuckRequestBuilder()
                 .url(downloadUrl)
-                .addHeader("Version", localScannerVersion);
+                .addHeader("Version", localScannerVersion)
+                .addHeader("Accept-Version", "ANY");
```

Comment:
Fixes IDETECT-4455. With the addition of this header, the /api/tools (new) scan-cli download request will behave as the old API did where upgrades/downgrades from any installed version to any other version was allowed.